src/client/helpers/parser.js

```diff
@@ -7,4 +7,27 @@ export function getFromMetadata(jsonMetadata, key) {
   return _.get(metadata, key);
 }
 
+const imgRegex = /<img[^>]+src="([^">]+)"/g;
```

Comment:
im not a regex expert but I think this matched this:

![screen shot 2018-01-15 at 1 50 29 am](https://user-images.githubusercontent.com/5067716/34930155-2c00369c-f996-11e7-9c69-c57d96b28e9a.png)

but I guess this would be a rare case, all code looks fine to me, not too sure on regex but if @bonustrack is fine with this then lgtm

Reply:
That's drawback of parsing HTML using Regexes. There is no simple regex that would handle all edge cases, it works as long as it's proper HTML. It breaks when body is just random content.

This would extract `src` properly though. I don't think it's worth it trying to make standard HTML parser using regex that handles non-standard "HTML".

Reply:
yeah makes sense, I've been trying to figure out ways on the side too, to try to extract html tags for other projects, its hard to do with just regex